notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,1495 @@
+{
```

Comment:
<p>I still think this section could use a simple, one sentence explanation of covariance, since the concept is very important for PCA in general (we're effectively trying to diagonalize this matrix -- but why? What will that tell us?). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>